library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -60,4 +60,12 @@ object Predef:
     inline def ne(inline y: AnyRef | Null): Boolean =
       !(x eq y)
 
+  extension [T](it: java.lang.Iterable[T])
+    def foreach(f: java.util.function.Consumer[T]): Unit =
+      it.forEach(f)
+
+  extension [T](s: java.util.stream.Stream[T])
+    def foreach(f: java.util.function.Consumer[T]): Unit =
```

Comment:
`foreach` needs to be `inline`, since `stdLibPatches/Predef.scala` has no runtime equivalent. Or else we need a PR against 2.13 Predef instead.